src/implementations/twig/components/table/table.story.js

```diff
@@ -10,30 +10,58 @@ import notes from './README.md';
 // Preserve original data
 const dataZebra = { ...dataDefault, zebra: true };
 
+const getArgs = () => ({
+  enhanced: false,
+});
+
+const getArgTypes = () => ({
+  enhanced: {
+    name: 'enhanced display',
+    type: { name: 'boolean' },
+    description:
+      'Enhanced display for mobile. This may depends on your needs and the data passed.',
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'Display',
+    },
+  },
+});
+
+const prepareData = (data, args) => {
```

Comment:
this can simply be `const prepareData = (data, args) => Object.assign(data, args);`, no..?